### PR TITLE
All Requests Permission Broken

### LINKED ISSUE #1906: All Requests Permission Broken
When creating a user, even after choosing to assign all permissions to that user, they are still unable to see the "All Requests" section of the requests page, nor are they able to see any tasks other than what have been assigned to them.

The only way it works is if the user is changed into a "Super Admin", but that's probably not a good idea.

### PR DESCRIPTION
**Pull request by milaendo**
closes #1906 

fixed the auth check so it now respects the "view all requests" permission.

to test: create a new user and give them "view all requests" permissions, login as that user and see all of the beautiful requests

### CODE REVIEW

File: resources/views/layouts/minimal.blade.php [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="shortcut icon" type="image/x-icon" href="https://pbs.twimg.com/profile_images/521685439820742657/8B2oQKmP_400x400.jpeg" />
     <meta name="csrf-token" content="{{ csrf_token() }}">
-    <title>@yield('title',__('Welcome')) - ProcessMaker')</title>
```
Comment (velkymx) [inline resources/views/layouts/minimal.blade.php]: Keep this

Comment (milaendo) [reply resources/views/layouts/minimal.blade.php]: its still there, just fixed the extra ')'. I can fix it is a different ticket if you would like!

Comment (velkymx) [reply resources/views/layouts/minimal.blade.php]: - ProcessMaker should always exist untranslated.

Comment (milaendo) [reply resources/views/layouts/minimal.blade.php]: gotcha!

**Pull request merged**